kplclient.go

```diff
@@ -110,21 +103,9 @@ func (c *KPLClient) processErrMessage() {
 
 func Read(conn net.Conn) (string, error) {
 	reader := bufio.NewReader(conn)
-	var buffer bytes.Buffer
-	for {
-		ba, isPrefix, err := reader.ReadLine()
-		if err != nil {
-			if err == io.EOF {
-				break
-			}
-			return "", err
-		}
-		buffer.Write(ba)
-		if !isPrefix {
-			break
-		}
-	}
-	return buffer.String(), nil
+	tp := textproto.NewReader(reader)
+	line, _ := tp.ReadLine()
+	return line, nil
```

Comment:
Any reason not to return the error here?
```suggestion
	return tp.ReadLine()
```

Reply:
I think I just missed it. Thanks.